pkg/config/config.go

```diff
@@ -65,6 +68,20 @@ func (w *IniWriter) Write(section string, keyValue map[string]string) error {
 		w.File.Section(section).Key(key).SetValue(value)
 	}
 
+	lock := fslock.New(w.ConfigFilepath)
```

Comment:
Should we add a suffix `.lock` to `ConfigFilepath`? I think fslock expects the file to be a tmp file, not a file we want to read/write in the same process after locking.